tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -539,7 +539,8 @@ def check_args(kwargs):
 
     if kwargs['extra_prefs']:
         # If a single pref is passed in as a string, make it a list
-        if type(kwargs['extra_prefs']) in (str, unicode):
+        from six import text_type
+        if isinstance(kwargs['extra_prefs'], text_type):
```

Comment:
Did we test this? How sure are we we always get unicode strings here?

Reply:
Oh that's a mistake indeed, we should use string_types there